tokens/index.js

```diff
@@ -156,7 +156,7 @@ const choices = {
   },
   fontFamily: {
     sans: ['Quicksand', 'Arial', 'sans-serif'],
-    // TODO: Add a monofont size to use in Tokens components
+    mono: ['Roboto Mono', 'monospace'],
```

Comment:
Creo que las comillas las podemos agregar desde aca
```suggestion
    mono: ['"Roboto Mono"', '"Courier New"', 'monospace'],
```